media/mathplayground/global.css

```diff
@@ -177,21 +184,3 @@ input:focus + .slider {
 .titlefont {
     font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
 }
-
-.top-title {
-    color: white;
-    font-size: smaller;
-    font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
-    user-select: none;
-    cursor: pointer;
-}
-
-.top-title:hover {
-    background-color: rgba(255, 0, 0, 0.3);
-}
-
-.chapterBox p,
-.chapterBox article,
-.objectBoxOuter {
-    color: white;
```

Comment:
This CSS rule needs to go back in - it affects the text in session mode:
![Screenshot_2022-12-08_14-15-01](https://user-images.githubusercontent.com/59292/206547042-c5682e7d-bb70-403e-9f60-812fff7ff018.png)